docs/advanced/volumes.md

```diff
@@ -150,8 +150,9 @@ The downsides:
 
 ### Using Unison volumes
 
-- [Follow instructions](http://docs.docksal.io/en/master/getting-started/env-setup-native/) to properly 
-configure Docksal for use with Docker for Mac. Make sure you have the most recent fin version installed.
+Unprecedented, native-like FS speed. See https://github.com/docksal/unison for details
```

Comment:
It's a good idea to provide a link to the user rather than just a URL. You want to give them a good clickable target. Something like:
[See unison project](https://github.com/docksal/unison) for details.